HookTest/cmd.py

```diff
@@ -19,10 +19,12 @@ def parse_args(args):
 
     parser.add_argument('-w', "--workers", type=int, help='Number of workers (processes) to be used', default=1)
 
-    parser.add_argument(
-        '-s', "--scheme",
-        help="Scheme to test. '-ignore' test will ignore RNG test but still inform which scheme is used",
-        default="tei", choices=("tei", "epidoc", "epidoc-ignore", "tei-ignore"))
+    parser.add_argument('-s', "--scheme",
+                        help="Scheme to test. '-ignore' test will ignore RNG test but still inform which scheme is used",
+                        default="tei",
+                        choices=("tei", "epidoc", "epidoc-ignore", "tei-ignore", "local", "remote", "auto"))
```

Comment:
I'd move away from choices, still accepting : 

- tei
- epidoc
- ignore
- auto [Default]

but it can also have a local path given, which would remove the need of `--rng`

Reply:
But, we need a new argument : `--guidelines` that would accept `["2.epidoc", "2.tei"]` which could be automatically induced for the moment when `tei` or `epidoc` is given as `scheme` for retrocompatibility purposes. It should default to `2.tei`.

Reply:
ChoicesOrPath("tei", "epidoc", "ignore", etc...)